src/components/implementation/no_interface/scheddev/fprr/sl_thd.h

```diff
@@ -22,6 +22,9 @@ struct sl_thd {
 	thdcap_t       thdcap;
 	tcap_prio_t    prio;
 	struct sl_thd *dependency;
+
+	cycles_t       period, wakeup_cycs;
+	int            wakeup_idx;
```

Comment:
comment for the `wakeup_idx`?  Not sure what it is.  I assume some sort of heap index?